howto-web-no-local-install.md

```diff
@@ -21,7 +21,7 @@ Let's see how to use OpenFisca on one of those services: [repl.it](https://repl.
     print parameters
     ```
 
-    > You can also import files (e.g. JSON files describing input [situations](http://openfisca.org/doc/openfisca-web-api/input-output-data.html#describing-the-situation)) by clicking on the `import or drop` button.
+    > You can also import files (e.g. JSON files describing input [situations](./openfisca-web-api/input-output-data.md#describing-the-situation)) by clicking on the `import or drop` button.
```

Comment:
Will it work with `.md`? Keep a `.html` link?

Reply:
> Add minor fix for `import or drop` and set it to `upload or drop`?

Reply:
I think in gitbook all relative links should be in `.md`, and are automatically compiled during the build process 🙂

Reply:
+1 to what @fpagnoux said 🙂 

Also, if you ever have any doubt about what the output will be, you are very encouraged to compile the book locally with `gitbook serve` 😃

Reply:
Regarding the suggested “minor fix”, I'd like to keep this PR focused on links update, not on updating a UI I don't know about 🙂